pages/docs/concepts/asyncapi-document/dynamic-channel-address.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Dynamic Channel Address
+weight: 80
+---
+
+Dynamic channel addresses are used to specify dynamic parts of a channel name, which becomes particularly useful when you want to use and reuse parameters in the channel name. It provides flexibility in how you structure your event-driven interfaces.
+
+Here is the diagram explaining dynamic channel address:
+
+```mermaid
+graph TD
+A[AsyncAPI]
+B[Dynamic Channel Address]
+D[Variable Usage in Channel Address]
+E[Adapted to Multiple APIs and Protocols]
+F[Reused Across Channels]
+
+style B fill:#47BCEE,stroke:#47BCEE;
+
+A --> B
+B --> D
+D --> E
+D --> F
+```
+
+The flowchart shows how dynamic channel addresses allow flexible event-driven interfaces by using and reusing parameters in channel names, adapting well to various APIs and protocols.
```

Comment:
We do not use the word "flowchart" for diagrams in our docs. Let's remember to be consistent and please update all mentions to say "diagram. "